File: Module 1/03.Methods/04.AppearanceCount.cs
using System;

class Program
{
    static void Main()
    {
        int size = int.Parse(Console.ReadLine());
        int[] myArray = new int[size];
        string[] elementsOfArray = Console.ReadLine().Split();
        for (int i = 0; i < elementsOfArray.Length; i++)
        {
            myArray[i] = Convert.ToInt16(elementsOfArray[i]);
        }
        
        int x = int.Parse(Console.ReadLine());
        int counter = 0;
        counter = AppearanceCount(counter, x, myArray);
        Console.WriteLine(counter);
    }

    static int AppearanceCount(int counter, int x, int[] SingleArray)
    {
        for (int i = 0; i < SingleArray.Length; i++)
        {
            if (SingleArray[i] == x)
            {
                counter++;
            }
        }
        return counter;
    }
}

File: Module 3/JavaScriptFundamentals/08.Functions/04.AppearanceCount.js
function AppearanceCount(input){
    var arraySize = input[0];
    var arrayOfNumbers = input[1].split(' ');
    var searchedValue = input[2];
    var counter = 0;

    for(var i = 0; i < arraySize; i++){
        if(arrayOfNumbers[i] == searchedValue){
            counter++;
        }
    }
    return arrayOfNumbers;
}